test/acceptance/features/createTodo.feature

```diff
@@ -0,0 +1,27 @@
+Feature: createTodo
+  As a user
+  I want to create a todo
+  So that I would keep track of my todo
+
+  Background:
+    Given user has browsed to the homepage
+
+  Scenario: user should be able to create a todo entering both subject and content
+    When the user creates a todo entering subject "subject" and content "content"
+    Then a new todo with subject "subject" and content "content" should be visible
+
+  Scenario: user should not be able to create a todo entering subject only
+    When the user creates a todo entering subject "subject" only
```

Comment:
```suggestion
    When the user tries to creates a todo entering subject "subject" only
```
Since this step would not be successful, it would be more readable this way
Do same for other similar steps